AGENTS.md

```
@@ -0,0 +1,155 @@
+pnpm build            # Production build (~6s)
+npx astro check       # TypeScript validation
+
+# E2E tests (dev server must be running)
+npx cypress run                                      # All tests
+npx cypress run --spec "tests/e2e/navigation.cy.ts" # Single test
+npx cypress open                                     # Interactive
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)